src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write your code here
+const button = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  button.addEventListener('click', resolve);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
```

Comment:
Consider using `Promise.race([promise1, promise2])` to handle only the first promise that settles. This will prevent both messages from being displayed if the button is clicked after the timeout.